packages/admin-page/src/footer.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Automattic\Jetpack\AdminPage;
+
+/**
+ * Create and Render the Jetpack Footer
+ */
+class Footer {
+
+	/**
+	 * Build and render the Jetpack page Footer
+	 *
+	 * @return string
+	 */
+	public function render() {
+
```

Comment:
```suggestion
```

Reply:
0dbe752ca (not yet pushed) removes this line.